cpp/bench/sg/fil.cu

```diff
@@ -97,8 +98,10 @@ class FIL : public RegressionFixture<float> {
     this->loopOnState(state, [this]() {
       // Dataset<D, L> allocates y assuming one output value per input row,
       // so not supporting predict_proba yet
-      ML::fil::predict(*this->handle, this->forest, this->data.y, this->data.X,
-                       this->params.nrows, false);
+      for (int i = 0; i < p_rest.predict_times; i++) {
```

Comment:
What times will be reported by the benchmark? Will the time taken by all 10 repetitions reported (which the user then must manually divide by 10)? Or will it be the time for a single repetition?

Reply:
time for 10 repetitions, to be manually divided by 10. Until there's an parameter added to MLCommon::Bench::Fixture, there's no way to do that automatically.

Reply:
Could you add a TODO for this?